f | tests.py | s | 2.4K | 54 | Claude Paroz | claude@2xlibre.net | 1335725835 |  | Fixed #18013 -- Use the new 'as' syntax for exceptions.  Thanks Clueless for the initial patch. Note that unittest has been purposely left out (external package only used by Python 2.6).
f | models.py | s | 648B | 17 | Claude Paroz | claude@2xlibre.net | 1339085327 |  | Fixed #18269 -- Applied unicode_literals for Python 3 compatibility.  Thanks Vinay Sajip for the support of his django3 branch and Jannis Leidel for the review.
f | __init__.py | s | 0B | 0 | Adrian Holovaty | adrian@holovaty.com | 1149638969 |  | Added Manager.get_or_create()  git-svn-id: http://code.djangoproject.com/svn/django/trunk@3092 bcc190cf-cafb-0310-a4f2-bffc1f526a37
